src/isBlank.js

```diff
@@ -0,0 +1,44 @@
+import * as R from 'ramda';
```

Comment:
We use explicit named imports in source code:
```suggestion
import { match, isEmpty } as R from 'ramda';
```